BBHSE/view.php

```diff
@@ -0,0 +1,286 @@
+<?php
+//by Art Eaton
+include_once("../../globals.php");
+?>
+<html><head>
+<? html_header_show();?>
+<link rel=stylesheet href="<?echo $css_header;?>" type="text/css">
```

Comment:
...and maybe use some .js (with the header stuff) to shore up these old nasty forms.